### ISSUE TITLE
Support running all tasks even on errors

### ISSUE THREAD

**Issue by xremming** [go-task/task#1853]
So in my use case I'm consolidating all different linters (among other things) to be ran with the Taskfile. See the snippet below for the general structure.

When running linters it would be nice to run _all_ of them always (the same is especially true with the for loop) and once they're all ran to return an error if any of them errored. I could work around this by using longer shell scripts but I would prefer not to.

I'm thinking something like a `run_all: always` parameter. The default for which could be the current behaviour of `stop-on-first-error`.

The `ignore_error` works otherwise but it also ignores the error code, meaning that when the linters are ran in CI it will not cause the run to fail.

```yaml
tasks:
  lint:
    deps: [lint:terraform:fmt, lint:terraform:tflint]

  lint:terraform:fmt:
    dir: infra
    cmd: "terraform fmt -check -recursive"

  lint:terraform:tflint:
    deps: [install:tflint]
    dir: infra
    vars:
      STACKS:
        sh: "echo stacks/*"
    cmd:
      for:
        var: STACKS
      cmd: "tflint --config=../../.tflint.hcl --chdir={{ .ITEM }}"
```

**Comment by andrewimeson** [comment]
This is probably a duplicate of #1246

I posted an earlier iteration of the workaround I use on that issue.

This feature would be really nice.

**Comment by xremming** [comment]
The suggestion in #1246 on how to handle it seems better. So a +1 for `failfast` from me.

**Comment by trulede** [comment]
The code suggests that:

- deps are run in parallel, there does not seem to be any mechanism which is killing tasks, it runs them all in parallel and returns the first failing task (of all the tasks that failed).
https://github.com/go-task/task/blob/6645a1f34ccc8f9305e137ded2a6493f05543f7d/task.go#L305
- if you use the -parallel flag and list the tasks at the CLI then those tasks will run in parallel, using the same mechanism as above 
https://github.com/go-task/task/blob/6645a1f34ccc8f9305e137ded2a6493f05543f7d/task.go#L133

@xremming  If you modify your example to something people can easily run (e.g. tasks with cmds exit(0) or exit(1) and some echos) and share the CLI command you use, it might be possible to clarify the behaviour.

**Comment by trulede** [comment]
Close as duplicate of #1246, which has a fix available in the latest version of Task.
